packages/amplication-client/src/Resource/code-view/FileExplorerNode.tsx

```diff
@@ -6,9 +6,16 @@ import { FileMeta } from "./CodeViewExplorer";
 export type FileExplorerNodeProps = {
   file: FileMeta;
   onSelect: (data: FileMeta) => void;
+  currentFolder: string;
+  isLoading: boolean;
 };
 
-export const FileExplorerNode = ({ file, onSelect }: FileExplorerNodeProps) => {
+export const FileExplorerNode = ({
+  file,
+  onSelect,
+  currentFolder,
```

Comment:
what are we using this prop for?

Reply:
It's used in conjunction with the isLoading prop to manage the display of loading placeholders for folders.

Reply:
I do not see where it is being used - you get it and pass it forward into the children of the same type but I don't see any code that uses this prop